File: toolkit/bean/SystemParam.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="systemParam")
public class SystemParam extends BaseApiBean {
  private Media media;
  private Security security;

  public Media getMedia() {
     return this.media;
  }

  public void setMedia(Media media) {
     this.media = media;
  }

  public Security getSecurity() {
     return this.security;
  }

  public void setSecurity(Security security) {
     this.security = security;
  }


}

File: CiscoPCCE.Toolkit/Bean/SystemParam.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlType("systemParam")]
    public class SystemParam : BaseApiBean
    {
        [XmlElement("media")]
        public required Media Media { get; set; }

        [XmlElement("security")]
        public required Security Security { get; set; }
    }
}